CacheManager.StackExchange.Redis.SelfCleaning/SelfCleaningRedisConfigurationBuilderExtensions.cs

```diff
@@ -0,0 +1,64 @@
+﻿using System;
+using CacheManager.Core;
+using CacheManager.Redis;
+using CacheManager.StackExchange.Redis.SelfCleaning.Timers;
+using StackExchange.Redis;
+
+namespace CacheManager.StackExchange.Redis.SelfCleaning
+{
+    public static class SelfCleaningRedisConfigurationBuilderExtensions
+    {
+        private const string DEFAULT_CONFIGURATION_KEY = "redis";
+        private const string CONFIGURATION_KEY_SUFFIX = "-self_cleaning";
```

Comment:
Maybe add comment  about those two keys, that they have no special meaning, and they exist because of the way Redis saves it's configuration (doctionary)?

Reply:
Added